packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -450,21 +453,18 @@ def get_raw_safe_transaction(  # pylint: disable=too-many-arguments,too-many-loc
             and max_priority_fee_per_gas is None
         ):
             tx_parameters.update(ledger_api.try_get_gas_pricing(old_price=old_price))
-        # note, the next line makes an eth_estimateGas call!
-
+        # note, the next line makes an eth_estimateGas call iff gas is not set!
         transaction_dict = w3_tx.buildTransaction(tx_parameters)
-        gas_estimate = (
-            ledger_api._try_get_gas_estimate(  # pylint: disable=protected-access
-                transaction_dict
+        if configured_gas != 1:
```

Comment:
We now either price exactly as configured or we use the estimated value. For the estimation, we need to be able to specify a fallback as the estimation can fail.